spotui/src/spotifyApi.py

```diff
@@ -170,8 +170,22 @@ def get_playlists(self):
             self.auth()
             if not self.token:
                 return []
-            playlists = self.client.current_user_playlists()
-            return list(map(self.__map_playlists, playlists["items"]))
+
+            # this grabs playlists spotify is trying to get the user to listen to
+            # playlists = self.client.user_playlists('spotify')
+
+            playlists = []
+            offset = 0
+
+            while True:
+                get_playlists = self.client.current_user_playlists(offset=offset)["items"]
+                playlists.extend(get_playlists)
+                if len(get_playlists) < 50:
+                    break
+                offset += 50
```

Comment:
What's the most amount you've tested with? It would be useful to know roughly how many times we can hammer the endpoint before we get rate-limited (you'll get a 429). Think we should check with like... 500 playlists? That should cover most people. 

I'll make some time to test this on Sunday and just hardcode it to 10 requests to check rate-limiting and impact on load.

Even if this is tolerable in terms of load etc, might be cool to lazy load these in the future (e.g. fetch an initial amount and then when the user gets like 75% of the way through scrolling them, load another batch etc). Would be much more complicated to implement and test though and this fixes the bug so it's one for the backburner. 

I'll leave this open until one of us has had a chance to test